.github/SUPPORT.md

```diff
@@ -0,0 +1,26 @@
+# Community & Support
+
+## Issue tracker
+
+Create a bug report on [our bug tracking
+system](https://github.com/Automattic/wp-calypso/issues/new) or check on the
+status of an existing bug in our [issue
+tracker](https://github.com/Automattic/wp-calypso/issues).
+
+## Developer Blog
+
+Check out the [Wordpress.com developer
+blog](https://developer.wordpress.com/blog/) for technical news.
+
+## Support
+
+If you are a developer who needs help with *WordPress.com APIs and related
+tools*, please [contact us](https://developer.wordpress.com/contact/).
+
+If you need help with your *WordPress.com site or account*, please [contact our
+happiness engineers](http://en.support.wordpress.com/contact/).
```

Comment:
The en isn't necessary here as it will default to the customer's language

Reply:
Hi @alisterscott I removed it. Was there due I checked it via incognito for not logged in user

![ss_2017-09-07-07-54-32](https://user-images.githubusercontent.com/454800/30148197-a287be20-93a2-11e7-843a-caee315be473.jpg)